src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    Square(String color, double side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.

Reply:
I don't pass expressions in the constructor. Look at the code again :(